docs/contribute.rst

```diff
@@ -499,6 +499,21 @@ variable to include additional file:
     override theme CSS, it is recommended to add ``!important`` flag.
 
 
+TLDR pages
+------------------------------------------------------------------
+
+tmt's tldr pages are maintained in the central `tldr-pages repository <https://github.com/tldr-pages/tldr>`_.
+To modify existing pages or add new ones, submit your changes directly to the `tldr-pages/tldr repository <https://github.com/tldr-pages/tldr>`_
+by following their `contribution guidelines <https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md>`_.
```

Comment:
I believe we follow a slightly different approach to including links in the text:

```
tmt's tldr pages are maintained in the central `tldr-pages`__ repository.
To modify existing pages or add new ones, submit your changes directly to the `tldr-pages/tldr repository` by following their `contribution guidelines`.

__ https://github.com/tldr-pages/tldr
__ https://github.com/tldr-pages/tldr
__ https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
```

It should make the paragraph contain just the text, making it a bit more readable.

BTW two links are the same, is that expected?

Reply:
Hmm, not sure if it's morr readable, but if it's how it's being done..  

Hmm, no I think I wanted to link the specific directory with the relevant tmt .md pages, but probably changed my mind because of the translated pages being elsewhere, so the guidelines are the place to go. 

@psss Might I ask the resident documentation god for edit here? :)

Reply:
Ok, done in d3c5369a :-)